### DISCUSSION TITLE
Upgraded to 3.11 - Issues

### DISCUSSION THREAD

**Discussion by jmartindg** [nuxt/nuxt#26403] [Questions] [answered]
Hello, I did an update from 3.9.3 to 3.11 version. After updating Nuxt successfully, the dev environment is having some issues. The whole project is not responding, there are no console errors, inspect element is not working, I can't click any user interfaces, I can't also redirect to another pages. Thank you.

**Comment by TheAlexLichter** [comment]
Very hard to tell without code or similar. Try to use the nightly channel versions and figure out which commit breaks the app ☺️

**Reply by jmartindg** [reply]
Here is my package.json file
```
{
  "name": "nuxt-app",
  "private": true,
  "type": "module",
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "autoprefixer": "^10.4.17",
    "nuxt": "^3.11.1",
    "postcss": "^8.4.33",
    "tailwindcss": "^3.4.1",
    "vue": "^3.4.14",
    "vue-router": "^4.2.5",
    "vuetify": "^3.5.11"
  },
  "dependencies": {
    "@mdi/font": "^7.4.47",
    "nuxi": "^3.11.1",
    "vite-plugin-vuetify": "^2.0.3"
  }
}

```

**Comment by ahmeddysf100** [comment]
same issue i upgrade from 3.10.3 to 3.11

**Reply by mubaidr** [reply]
Did you try to recreate `node_modules` and `package-lock` file?

Please try:

```bash
npx nuxi upgrade --force
```
or
```bash
pnpm nuxt upgrade --force
```

**Comment by mubaidr** [comment, marked as answer]
The issue has been identified; it is the component library at fault here: https://github.com/nuxt/nuxt/issues/26488#issuecomment-2022123332
